cranelift/codegen/src/isa/aarch64/inst/emit.rs

```diff
@@ -2136,7 +2071,8 @@ impl MachInstEmit for Inst {
                 sink.put4(0xd4200000);
             }
             &Inst::Udf { trap_info } => {
-                let (srcloc, code) = trap_info;
+                let srcloc = state.cur_srcloc();
+                let code = trap_info;
                 sink.add_trap(srcloc, code);
```

Comment:
```suggestion
                sink.add_trap(srcloc, trap_info);
```
This avoids unnecessary `let code = ...`; possibly rename the `Udf` field to `trap_code`.

Reply:
Good point, thanks!